internal/blocksync/reactor.go

```diff
@@ -61,6 +62,7 @@ type Reactor struct {
 	store         sm.BlockStore
 	pool          *BlockPool
 	blockSync     bool
+	myAddr        crypto.Address
```

Comment:
```suggestion
	validatorAddr        crypto.Address
```

Reply:
But we're not sure we're a validator...

Reply:
But it is our validatorAddress, right, not network address.

Reply:
It could be empty for a full node and we are still good

Reply:
I still think `validatorAddr` is misleading here

Reply:
How about `localAddr`?

Reply:
Where are looking at the validator set where we expect to find a ... validator address.

But this is naming of variables, please feel free to choose.